covid/transform.py

```diff
@@ -513,14 +533,61 @@ def transform_covidtracking_data(df):
         )
 
     # Drop American Samoa because it's not reporting data
-    df = df.loc[df[STATE_SOURCE_FIELD] != "American Samoa",]
+    df = df.loc[
+        df[STATE_SOURCE_FIELD] != "American Samoa",
+    ]
 
     # Copy state values into column called "State" instead of "state".
     df[STATE_FIELD] = df[STATE_SOURCE_FIELD]
 
     return df
 
 
+def transform_cdc_data(cdc_current_df, cdc_historical_df):
+    # Add date to index
+    cdc_df = pd.concat([cdc_current_df, cdc_historical_df], axis=0)
```

Comment:
Got it -- so based on this and your very helpful comment, for now, we'll only have data from the first time we ran @BriceWolfgang's scraper (6/10)? 

If I'm understanding that correctly, maybe we can create a backlog issue to convert the CSV we found of bed data and upload it once to this sheet.

Reply:
Yes that sounds like a plan!